front/app/containers/Admin/reporting/components/ReportBuilder/Templates/ProjectTemplate.tsx

```diff
@@ -49,7 +49,8 @@ const ProjectTemplate = ({ reportId, projectId }: Props) => {
   );
 
   const hasPhases =
-    project.data.attributes.process_type === 'continuous' && phases.data.length > 0;
+    project.data.attributes.process_type === 'continuous' &&
+    phases.data.length > 0;
```

Comment:
Not sure where this is coming from... 🧐 Probably an artifact of the conflict resolution with the master branch.